docs/source/workflows/predictors.rst

```diff
@@ -134,8 +137,167 @@ The following example demonstrates how to create an :class:`~citrine.informatics
+| isotonic saline   | 0.99           | 0.01          | 1.01           |
++-------------------+----------------+---------------+----------------+
+| water             |                |               | 1.0            |
++-------------------+----------------+---------------+----------------+
+| salt              |                |               | 2.16           |
++-------------------+----------------+---------------+----------------+
+
+There are two mixtures, hypertonic and isotonic saline formed by mixing water and salt together in different amounts.
+(Note, water and salt are leaf ingredients; and, hence these rows do not contain quantity data.)
+Mixtures are defined by a map from ingredient id to quantity, so this predictor would form 2 mixtures with recipes:
+
+.. code:: python
+
+    # hypertonic saline
+    {'water': 0.93, 'salt': 0.07}
+
+    # isotonic saline
+    {'water': 0.99, 'salt': 0.01}
+
+Ingredients may be given 0 or more labels.
+Labels provide a way to group or distinguish one or more ingredients and can be used to featurize mixtures (discussed in the next section).
+The same label may be given to multiple ingredients, and a single ingredient may be given multiple labels.
+Labels are specified using a map from each label to a list of all ingredients that should be given that label.
+Anytime a recipe contains a non-zero amount of labeled ingredient, the ingredient is assigned the label.
+For example, we may wish to label ``water`` as a solute and ``salt`` as a solvent.
+These labels are specified via:
+
+.. code:: python
+
+    labels = {'solvent': ['water'], 'solute': ['salt']}
+
+The following example illustrates how an :class:`~citrine.informatics.predictors.IngredientsToSimpleMixturePredictor` is constructed for the saline example.
+
+.. code:: python
+
+    from citrine.informatics.descriptors import FormulationDescriptor, RealDescriptor
+    from citrine.informatics.predictors import IngredientsToSimpleMixturePredictor
+
+    # create a descriptor to hold simple mixtures
+    formulation = FormulationDescriptor('simple mixture')
+
+    # create descriptors for each ingredient quantity
+    water_quantity = RealDescriptor('water quantity', 0, 1)
+    salt_quantity = RealDescriptor('salt quantity', 0, 1)
+
+    IngredientsToSimpleMixturePredictor(
+        name='Ingredients to simple mixture predictor',
+        description='Constructs a mixture from ingredient quantities',
+        output=formulation,
+        # map from ingredient id to its quantity
+        id_to_quantity={
+            'water': water_quantity,
+            'salt': salt_quantity
+        },
+        # label water as a solvent and salt a solute
+        labels={
+            'solvent': ['water'],
+            'solute': ['salt']
+        }
+    )
+
+Generalized mean property predictor
+-----------------------------------
+
+Often, properties of a mixture are proportional to the properties of it's ingredients.
```

Comment:
Presumably the change I made in #333 against this line will be applied over this PR?

Reply:
Yes, it will be updated once we merge #333.